client/plrdlg_common.cpp

```diff
@@ -266,11 +266,11 @@ static QString col_idle(const struct player *plr)
 {
   int idle;
   static char buf[100];
-
-  if (plr->nturns_idle > 3) {
-    idle = plr->nturns_idle - 1;
-  } else {
+  // nturns_idle >= 0
```

Comment:
I think this is old code? If so, I would prefer if you could remove it. If it's a comment, I don't understand what it's trying to say (and I don't think a comment is necessary, but fine to add one as well).

Reply:
If you're talking about line 269:
// nturns_idle >= 0
Then it's not the old code but a commentary that the value isn't negative (as it's declared int and not unsigned int in the player struct), after checking the usages of nturns_idle and seeing that it's never being decreased but only increased or set to 0.
If you referring to something else then excuse me for misunderstanding.

Reply:
Aah ok, I see now that the code would behave weirdly for negative values. Sounds good then!